webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/smartling/SmartlingPluralFix.java

```diff
@@ -0,0 +1,34 @@
+package com.box.l10n.mojito.service.thirdparty.smartling;
+
+import com.box.l10n.mojito.service.tm.search.TextUnitDTO;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.box.l10n.mojito.android.strings.AndroidPluralQuantity.OTHER;
+
+public final class SmartlingPluralFix {
+
+    private SmartlingPluralFix() {
+        throw new AssertionError("Do not instantiate");
+    }
+
+    public static List<TextUnitDTO> fixTextUnits(List<TextUnitDTO> textUnits) {
```

Comment:
on top of my head there were some locales explicitly mentioned in the code for which the fix should be done. I don't see it here, any logic behind that?

Reply:
The list of locales to be plural fixed will be passed through the command line, with the `smartling-plural-fix` option. E.g: `mojito thirdparty-sync -r repo -p projectId -o "smartling-pluralfix=pt-BR,pt-PT"`, so if we receive locales in the command line, we apply the fix for them, otherwise, nothing is done.